mkdocs.yml

```diff
@@ -0,0 +1,102 @@
+site_name: SciLifeLab Architecture Board
+site_url: https://scilifelab.github.io/architecture/
+site_description: Architecture decisions and guidelines for SciLifeLab infrastructure
+site_author: SciLifeLab Architecture Board
+
+repo_name: scilifelab/architecture
+repo_url: https://github.com/scilifelab/architecture
+
+theme:
+  name: material
+  logo: assets/logo.png
+  favicon: assets/favicon.png
+  font:
+    text: Open Sans
+    code: Fira Code
+  icon:
+    repo: fontawesome/brands/github
+  palette:
+    - media: "(prefers-color-scheme: light)"
+      scheme: default
+      primary: teal
+      accent: lime
+      toggle:
+        icon: material/brightness-7
+        name: Switch to dark mode
+    - media: "(prefers-color-scheme: dark)"
+      scheme: slate
+      primary: teal
+      accent: lime
+      toggle:
+        icon: material/brightness-4
+        name: Switch to light mode
+  features:
+    - navigation.instant
+    - navigation.tracking
+    - navigation.tabs
+    - navigation.top
+    - navigation.footer
+    - search.suggest
+    - search.highlight
+    - content.code.copy
+
+extra_css:
+  - stylesheets/extra.css
+
+markdown_extensions:
+  - admonition
+  - pymdownx.details
+  - pymdownx.emoji:
+      emoji_index: !!python/name:material.extensions.emoji.twemoji
+      emoji_generator: !!python/name:material.extensions.emoji.to_svg
+  - pymdownx.superfences:
+      custom_fences:
+        - name: mermaid
+          class: mermaid
+          format: !!python/name:pymdownx.superfences.fence_code_format
+  - pymdownx.highlight:
+      anchor_linenums: true
+  - pymdownx.inlinehilite
+  - tables
+  - attr_list
+  - toc:
+      permalink: true
+
+plugins:
+  - search
+
+nav:
+  - Home: index.md
+  - Architecture Decisions:
+      - decisions/0001-record-architecture-decisions.md
+      - decisions/0002-use-cc-by-sa-4-0-license.md
+      - decisions/0003-use-single-process-approach-for-architecture-requests.md
+      - decisions/0004-use-mkdocs-material-for-documentation-site.md
```

Comment:
Can we automatically create this? If not, I would suggest adding a checkbox to the ADR PR template:
> - [ ] I have listed this ADR in the `mkdocs.yml` nav section